triage-process.md

```diff
@@ -0,0 +1,60 @@
+# High Level Proposal:
+1. Define and publish a process to handle issues in a structured manner.
+    1. Issue Handling Process
+    1. Issue Stages
+    1. Roles
+    1. SLA
+
+## Process Definition
+
+### Issue Handling Process
+1. All incoming issues and PR’s should be assessed. These are items that a Triager would look at on a daily basis
```

Comment:
Nit: replace these smart quotes everywhere

Reply:
This was done. I had copy pasted what you had shared in the gdoc.